src/main/java/ratpack/pac4j/internal/RatpackSessionStore.java

```diff
@@ -19,17 +21,18 @@ public RatpackSessionStore(final SessionData session) {
     }
 
     @Override
-    public String getOrCreateSessionId(final RatpackWebContext context) {
-        return session.getSession().getId();
+    public Optional<String> getSessionId(final WebContext context, boolean createSession) {
+        return Optional.of(session.getSession().getId());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
I feel like there should be a way to do this that doesn't require suppressing this warning, but I can't find it.

@jthurne Any thoughts?

Reply:
Yeah, I can't think of anything better that would work here as well.